maketest.sh

```diff
@@ -29,3 +33,24 @@ else
 	$MAKE $@
 fi
 
+# For now (while this code is being prototyped) I only want this to be
+# done while running in jenkins, to avoid a potentially destabilising
+# change if people are running other java processes on their local system
```

Comment:
This reminds me that we need to split out what is pure build and test scripts vs what is Adopt Build farm functionality.  X-team discussion required methinks :-)